api.md

```diff
@@ -468,6 +549,46 @@ CREATE INDEX ON conditions(time, location) USING brin
   WITH (timescaledb.transaction_per_chunk);
 ```
 
+## delete_data_node() [](delete_data_node)
+
+This function will remove the data node locally. This will *not*
+affect the remote database in any way, it will just update the local
+index over all existing data nodes.
+
+The data node will be detached from all hypertables that are using
+it if permissions and data integrity requirements are satisfied. For
+more information, see [`detach_data_node`](#detach_data_node).
+
+#### Errors
+
+An error will be generated if the data node cannot be detached from
+all attached hypertables.
+
+#### Required Arguments [](delete_data_node-required-arguments)
+
+| Name        | Description            |
+| ----------- | -----------            |
+| `node_name` | Name of the data node. |
+
+#### Optional Arguments [](delete_data_node-optional-arguments)
+
+| Name        | Description                                           |
+|-------------|-------------------------------------------------------|
+| `if_exists` | Prevent error if the data node does not exist. Defaults to false. |
+| `cascade`   | Cascade the delete so that all objects dependent on the data node are deleted as well. Defaults to false. |
```

Comment:
I have no idea what the dependent objects. Will it be important for users to know?